projects/dot-rules/src/lib/rule-engine.ts

```diff
@@ -46,7 +46,8 @@ const I8N_BASE = 'api.sites.ruleengine';
     </div>
   </div>
   <div class="cw-rule-engine__empty" *ngIf="!rules.length">
-      <span>{{rsrc('inputs.want.to.add.rules') | async}}</span>
+      <i class="material-icons">tune</i>
```

Comment:
We have a dot-icon component: https://github.com/dotCMS/core-web/blob/master/src/app/view/components/_common/dot-icon/dot-icon.component.html#L1